src/server/handlers/join.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2022-2023 Protocol Labs
```

Comment:
I think we should group handles by topical modules. In this case, add, join, kill and leave should be part of a subnet or manager module, WDYT?

Reply:
updated and placed in a separate mod.